sql/updates/latest-dev.sql

```diff
@@ -250,3 +250,41 @@ CREATE FUNCTION @extschema@.show_chunks(
      created_after          "any" = NULL
  ) RETURNS SETOF REGCLASS AS '@MODULE_PATHNAME@', 'ts_chunk_show_chunks'
  LANGUAGE C STABLE PARALLEL SAFE;
+
+-- A version of makeaclitem that accepts a comma-separated list of
+-- privileges rather than just a single privilege. This is copied from
+-- PG16, but since we need to support earlier versions, we provide it
+-- with the extension.
+CREATE FUNCTION _timescaledb_functions.makeaclitem(regrole, regrole, text, bool)
+RETURNS AclItem AS '@MODULE_PATHNAME@', 'ts_makeaclitem'
+LANGUAGE C STABLE PARALLEL SAFE;
+
+-- Repair relations that have relacl entries for users that do not exist in pg_authid
```

Comment:
I am wondering if we should keep this function exported and provide it to users. As you write in the commit, chunks can not be created if the ACLs are broken and it might be good to be able to call this function in these situations.

Reply:
I had that in the first version of the PR but have got comments before that we should not do that. I agree that it might be good to have a set of functions of this kind both for testing purposes and for helping users that end up in this situation in other ways.